docs/sources/docker-hub/repos.md

```diff
@@ -142,3 +142,37 @@ similar to the example shown below.
 Webhooks allow you to notify people, services and other applications of
 new updates to your images and repositories.
 
+### Webhook chains
+
+Webhook chains allow you to chain calls to multiple services. After clicking the
+"Add webhook" button, simply add as many URLs as necessary in your chain.
+
+The first webhook in a chain will be called after a successful push. Subsequent URLs will be contacted after the callback has been validated.
+
+#### Validating a callback
+
+In order to validate a callback in a webhook chain, you need to
+
+1. Retrieve the `callback_url` value in the request's JSON payload.
+1. Send a POST request to this URL containing a valid JSON body.
+
+> **Note**: A chain request will only be considered complete once the last
+> callback has been validated.
+
+#### Callback JSON data
+
+Recognized parameters in callback data are as follow:
+
+* `state` (required): Accepted values are `success`, `failure` and `error`. If the state isn't `success`, the webhook chain will be interrupted.
+* `description`: A string containing miscellaneous information that will be available on the Docker Hub. Maximum 255 characters.
+* `context`: A string containing the context of the operation. Can be retrieved on the Docker Hub. Maximum 100 characters.
```

Comment:
@fredlf Since these are webhooks it could be anything, so Context lets the user say what this webhook is related too, since these will be used by 3rd party services, it would be their name or something meaningful to the user who set it up, could be "jenkins", "deployer",  other service, etc.